pkg/workceptor/kubernetes.go

```diff
@@ -203,7 +206,12 @@ func (kw *kubeUnit) createPod(env map[string]string) error {
 			return kw.clientset.CoreV1().Pods(ked.KubeNamespace).Watch(kw.ctx, options)
 		},
 	}
-	ev, err := watch2.UntilWithSync(kw.ctx, lw, &corev1.Pod{}, nil, podRunningAndReady)
+
+	ctxWithTimeout := kw.ctx
+	if kw.podPendingTimeout != time.Duration(time.Duration.Seconds(0)) {
```

Comment:
I think this can just be `!= time.Duration(0)`

Reply:
I have made the required changes.